backend/BackendMIIT/src/main/java/com/example/BackendMIIT/parser/util/ParserUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.example.BackendMIIT.parser.util;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+
+public class ParserUtil {
+
+    public static Document getDocumentByUrl(String url) throws IOException {
+        return Jsoup.connect(url).get();

```

Comment:
add maxBodySize() to read all of html

Reply:
Used the maxBodySize(0) and SneakyThrows annotation